ui/src/app/views/sign-in-flow-template/component.tsx

```diff
@@ -49,7 +54,19 @@ const SignPageTemplateReact = withWindowSize()(
       super(props);
       this.state = {
         currentStep: 'login',
-        invitationKey: ''
+        invitationKey: '',
+        profile: {
+          // Note: We abuse the "username" field here by omitting "@domain.org". After
+          // profile creation, this field is populated with the full email address.
+          username: '',
+          dataAccessLevel: DataAccessLevel.Unregistered,
+          givenName: '',
+          familyName: '',
+          contactEmail: '',
+          currentPosition: '',
+          organization: '',
+          areaOfResearch: ''
+        }
```

Comment:
Is the profile needed here? It doesn't look like it's used anywhere.

Reply:
not yet, but by the time we finish the full flow.